units/Legion/Legion EvoCom/legcomlvl3.lua

```diff
@@ -371,6 +371,9 @@ return {
 				weapontimer = 5,
 				weapontype = "MissileLauncher",
 				weaponvelocity = 650,
+				customparams = {
+					stockpilelimit = 3,
+				},
```

Comment:
Unused weapon but has stockpile configuration so should have limit, or could be deleted entirely

Reply:
Removed unused weapondef